### PR TITLE
Tweaks and additions to camera page

### PR DESCRIPTION
**Pull request by langsmith**
This pr makes adjustments and additions to https://www.mapbox.com/android-docs/maps/overview/camera/ . Added a code snippet to explain some of the writing.

screencapture-localhost-8080-android-docs-maps-overview-camera-2018-11-30-16_31_06

### CODE REVIEW

File: src/pages/maps/overview/camera.md [hunk 1]
```diff
@@ -81,7 +81,22 @@ Check out [this Mapbox blogpost](https://www.mapbox.com/blog/designing-maps-for-
   />
 }}
 
-The `MapboxMap` class in the Maps SDK has several methods to change the camera's position. Each camera movement API takes in a `CameraUpdate` (built using `CameraUpdateFactory`) which is how you provide the new camera position information. Camera update factory can build several different `CameraUpdate` objects including a `newLatLngZoom()`, `zoomBy()`, `newLatLngBounds()`, and several more. One particularly interesting `CameraUpdate` is `newCameraPosition()` which is how you'd pass in a built `CameraPosition`.
+The `MapboxMap` class in the Maps SDK has several methods to change the camera's position. Each camera movement API takes in a `CameraUpdate` object. You should use the `CameraUpdateFactory` class to to provide the new camera position information. `CameraUpdateFactory` can build several different `CameraUpdate` objects including a `newLatLngZoom()`, `zoomBy()`, `newLatLngBounds()`, and several more. A straightforward method in `CameraUpdate` is `newCameraPosition()` which is how you'd pass in a built `CameraPosition`.
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/camera.md]: >You should use the `CameraUpdateFactory` class to ~~to~~ provide the new camera position information. `CameraUpdateFactory` can build several different `CameraUpdate` objects including a `newLatLngZoom()`, `zoomBy()`, `newLatLngBounds()`, and several more.

double `to` 👯

File: src/pages/maps/overview/camera.md [hunk 2]
```diff
@@ -145,7 +160,7 @@ mapboxMap.animateCamera(CameraUpdateFactory.newLatLngBounds(latLngBounds, 10));
   />
 }}
 
-The `setLatLngBoundsForCameraTarget` method in the `MapboxMap` class can limit the map camera to any area of the world that you'd like. If you feed the `LatLngBounds` object a minimum of two `LatLng` objects/coordinates, a _invisible_ square will automatically be created restricting the camera to the region.
+The `setLatLngBoundsForCameraTarget` method in the `MapboxMap` class can limit the map camera to any area of the world that you'd like. If you feed the `LatLngBounds` object a minimum of two `LatLng` objects/coordinates, a _invisible_ square will automatically be created to restrict the camera to the region.
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/camera.md]: Oh. `square` or `rectangle`? 📐

File: src/pages/maps/overview/camera.md [hunk 3]
```diff
@@ -81,7 +81,22 @@ Check out [this Mapbox blogpost](https://www.mapbox.com/blog/designing-maps-for-
...
+
+java={`
+mapboxMap.animateCamera(CameraUpdateFactory.newCameraPosition(position), millisecondSpeed);
+`}
+
+kotlin={`
+mapboxMap.animateCamera(CameraUpdateFactory.newCameraPosition(position), millisecondSpeed)
+`}
+/>
+}}
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/camera.md]: Looks like these are identical -- is that correct? Just wanted to double-check! 😅

File: src/pages/maps/overview/camera.md (hunk 1 above)
Comment (langsmith) [reply src/pages/maps/overview/camera.md]: Fixed

File: src/pages/maps/overview/camera.md (hunk 3 above)
Comment (langsmith) [reply src/pages/maps/overview/camera.md]: Not supposed to be identical 🐸 Fixed

File: src/pages/maps/overview/camera.md (hunk 2 above)
Comment (langsmith) [reply src/pages/maps/overview/camera.md]: Rectangle 🤓  Fixed

**Pull request merged**